tutorials/fmc/fmc.yml

```diff
@@ -0,0 +1,16 @@
+name: fmc
+subscribe:
+- name: motor_command
+  mob: 1
+publish:
+- name: fmc
+  id: 0xC
+  freq_hz: 50
+  signals:
+    - name: adc_value
```

Comment:
I believe this was supposed to be a temperature sensor, so I recommend giving it a more descriptive name.